jest/front/jest.setup.js

```diff
@@ -1,11 +1,25 @@
 require('@testing-library/jest-dom/extend-expect');
 const { matchers } = require('@emotion/jest');
 
+jest.mock('next-i18next');
+jest.mock('next/router');
+
+const { useTranslation } = require('next-i18next');
+
 const {
   mswServer,
 } = require('./../../libs/front/components/src/test/msw/mswServer');
 
-beforeAll(() => mswServer.listen({ onUnhandledRequest: 'error' }));
+beforeAll(() => {
+  useTranslation.mockImplementation((namespace) => ({
+    t: jest
+      .fn()
+      .mockImplementation((key) =>
+        Array.isArray(namespace) ? `${key}` : `${namespace}:${key}`
+      ),
+  }));
```

Comment:
is it better than having a `next-i18next` folder in `src/__mocks__` ?
This is how it was handled before + in Valentin's PR